client/packages/invoices/src/StockOut/utils.tsx

```diff
@@ -344,7 +344,7 @@ export const PackQuantityCell = (props: CellProps<DraftStockOutLine>) => (
     {...props}
     max={props.rowData.stockLine?.availableNumberOfPacks}
     id={getPackQuantityCellId(props.rowData.stockLine?.batch)}
-    min={1}
+    min={0}
```

Comment:
I KNOW side quest bad BUT:
  - such tiny change 🥺 
  - This input is only used by these two modals (Prescription and OS edit line)
  - was preventing you from being able to clear an issue quantity once you set one

Reply:
Ah this has actually been raised as a separate bug: #3834